src/algebraic_geometry/prime_spectrum/basic.lean

```diff
@@ -406,26 +406,25 @@ lemma vanishing_ideal_closure (t : set (prime_spectrum R)) :
 zero_locus_vanishing_ideal_eq_closure t ▸ (gc R).u_l_u_eq_u t
 
 lemma vanishing_ideal_anti_mono_iff {s t : set (prime_spectrum R)}
-  (hs : is_closed s) (ht : is_closed t) :
-  s ⊆ t ↔ vanishing_ideal t ≤ vanishing_ideal s :=
+  (ht : is_closed t) : s ⊆ t ↔ vanishing_ideal t ≤ vanishing_ideal s :=
```

Comment:
It suffices to assume `t` is closed in this lemma; as for the next lemma (`strict_anti`), the forward direction works if `s` is closed, and the backward direction needs that `t` is closed.

By the way, I think all `anti_mono` should be replaced simply by `anti`, because we use "antitone" as the opposite of "monotone" and "strict_anti` as the opposite of "strict_mono". But the change is gonna have a large diff, and [mv_polynomial.vanishing_ideal_anti_mono](https://leanprover-community.github.io/mathlib_docs/ring_theory/nullstellensatz.html#mv_polynomial.vanishing_ideal_anti_mono) and [projective_spectrum.vanishing_ideal_anti_mono](https://leanprover-community.github.io/mathlib_docs/algebraic_geometry/projective_spectrum/topology.html#projective_spectrum.vanishing_ideal_anti_mono) would better be changed simultaneously to maintain consistency, so let's defer to another PR.